packages/lesswrong/components/review/PostsItemReviewVote.tsx

```diff
@@ -39,12 +54,13 @@ const PostsItemReviewVote = ({classes, post}: {classes:ClassesType, post:PostsLi
   if (!canNominate(currentUser, post)) return null
 
   const displayVote = indexToTermsLookup[newVote || post.currentUserReviewVote]?.label
+  const nominationsPhase = getReviewPhase() === "NOMINATIONS"
 
   return <div onMouseLeave={() => setAnchorEl(null)}>
 
-    <LWTooltip title={<div>Nominate this post by casting a preliminary vote.</div>} placement="right">
-      <div className={classes.button} onClick={(e) => setAnchorEl(e.target)}>
-        {displayVote || "Vote"}
+    <LWTooltip title={`${nominationsPhase ? "Nominate this post by casting a preliminary vote" : "Update your vote"}`} placement="right">
```

Comment:
Changes the tooltip during Review Phase. Adds some styling that helps distinguish cast-vote-weights from other random numbers on PostsItems.